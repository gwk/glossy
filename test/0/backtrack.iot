{
  'args': [
    'a b ab aab',
    'aa aaa',
  ],
}
