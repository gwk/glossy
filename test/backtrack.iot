{
  'compile': ['$PROJ/legs/__main__.py test/$NAME.legs -test -output $NAME.swift'],
  'cmd': './$NAME.swift',
  'args': [
    'a aa b bb abc aabbcc',
    'ab aabb',
  ],
}