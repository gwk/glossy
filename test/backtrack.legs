space: \_+
a+
b+
a+b+c+
