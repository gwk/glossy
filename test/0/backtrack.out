
arg1: 'a b ab aab'
arg1:1:1-2: `a`
a b ab aab
~
arg1:1:2-3: space
a b ab aab
 ~
arg1:1:3-4: a_star_b
a b ab aab
  ~
arg1:1:4-5: space
a b ab aab
   ~
arg1:1:5-7: a_star_b
a b ab aab
    ~~
arg1:1:7-8: space
a b ab aab
      ~
arg1:1:8-11: a_star_b
a b ab aab
       ~~~

arg2: 'aa aaa'
arg2:1:1-3: incomplete
aa aaa
~~
arg2:1:3-4: space
aa aaa
  ~
arg2:1:4-7: incomplete
aa aaa
   ~~~
